.gitignore

```diff
@@ -22,6 +22,7 @@ package-lock.json
 # composer
 vendor
 composer.js
+composer.json
```

Comment:
I don't think this should be here, maybe in .gitattributes for export ignore? cc @ideadude

Reply:
Moved this to .gitattributes